Flagsmith.FlagsmithClient/Flag.cs

```diff
@@ -0,0 +1,52 @@
+using Newtonsoft.Json;
+using FlagsmithEngine.Feature.Models;
+using FlagsmithEngine.Identity.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Flagsmith
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Flag
+    {
+        public Flag() { }
+        public Flag(Feature feature, bool enabled, string value)
+        {
+            this.Feature = feature;
+            this.Enabled = enabled;
+            this.Value = value;
+        }
+        [JsonProperty("featureId")]
+        protected int FeatureId;
+        [JsonProperty("feature")]
+        protected Feature Feature = null;
```

Comment:
As per the discussion on your previous PR, we should simplify this to just the string `featureName`. I'd like to keep the `Flag` class as slim as possible.

Reply:
working on it , as i would need to write custom serialization for that.